models/api-models.js

```diff
@@ -0,0 +1,25 @@
+// const fs = require('fs');
+
+// exports.selectAPI = () => {
+//   console.log("haldhkajfhsjkhlkfjdhk")
+//   return fs.readFile("./endpoints.json", "utf-8", (err, data) => {
+//     if (err) {
+//       console.log(err);
+//       return err;
+//     }
+//     else{
+//     console.log("hsg;ldgkfjhaduliihliausdfhi")
+//     const endpoints1 = JSON.parse(data);
+//     console.log(endpoints1)
+//     return endpoints1
+//     }
+//   });
+  
+// }
+
+const fs=require('fs/promises')
```

Comment:
You're referencing a file that currently "doesn't" exist, you've not committed this file but have created a way to access it. Returning the endpoints.json does not need a model as you can send the json in the controller.